### ISSUE TITLE
My friends cannot join the server

### ISSUE THREAD

**Issue by FlyingFryingPan** [OgarProject/Ogar#658]
I set up port forwarding correctly and checked that the ports are open, but it still does not work.

I am using the correct format for the link, and I was able to connect to the server on my own computer
http://agar.io/?ip=[ip adress]:[port]

But maybe I'm using the wrong ip address? 
I tried my local IP address, my external IP address, and 127.0.0.1 (as per the instructions on the readme file) but my friend couldn't connect to any of them. I was able to to connect to all of those except the external IP.

Please help :(

**Comment by NatsuTheGreat** [comment]
Yes you are using the wrong IP. You need to use the IP address that you find [here](ipchicken.com) Not any local IP addresses.

**Comment by NatsuTheGreat** [comment]
And if you cannot connect to your external IP, your either got the wrong external IP, or did setup port forwarding correctly. I have an issue myself, where as I cannot connect to my server, but others can. So they connect through external IP, and I have to connect through localhost. I really want this solved...

**Comment by GhostShot3** [comment]
@NatsuTheGreat Hello,you here ?

**Comment by NatsuTheGreat** [comment]
Yeah.

**Comment by PLASMAchicken** [comment]
You need to check if the Ports are open too http://www.yougetsignal.com/tools/open-ports/
